Randomizer.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+
+def randomly(list_of_students):
+    student_dict_reverted = {}
+    for elem in list_of_students:
+        student_dict_reverted[elem] = 0

```

Comment:
Why this is called `reverted`?

Reply:
Remained from the old code, already fixed.